modules/nf-core/summarizedexperiment/summarizedexperiment/tests/unify.config

```diff
@@ -0,0 +1,7 @@
+process {
+
+    withName: SUMMARIZEDEXPERIMENT_SUMMARIZEDEXPERIMENT {
```

Comment:
Wouldn't we need a different set of names for the gene and transcript objects?

Reply:
Still only one name specification?

To be clear, the transcript object has a different number of assays, needing a different set of names. So you should be using the aliased selectors (not `SUMMARIZEDEXPERIMENT_SUMMARIZEDEXPERIMENT`), and specifing this twice.

Reply:
This is the config for a test in the module. I made those changes in the config of the subworkflow.

Reply:
Ahh, quite right, apologies, clearly the coffee hasn't kicked in ;-)

Reply:
But, since this is going to be essential for operation of this subworkflow now, could you add an example config? See [this example](https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/fastq_align_star/nextflow.config) for another subworkflow.

Reply:
Yes, that's what I meant! It is here already here :)
https://github.com/nf-core/modules/blob/1bd155bf8a5bef8583bc252c0c14a6bd02696c4d/subworkflows/nf-core/quantify_pseudo_alignment/tests/nextflow.config#L2-L10

Reply:
So, this powers the tests. The example I linked sits one level higher, next to the main.nf, to show potential users how they need to configure the subworkflow if they use it.

Reply:
Apologies, I understood now.